driver-db/ens160.toml

```diff
@@ -0,0 +1,34 @@
+manifest_version = "0.0.1"
+
+[meta]
+categories = ["Sensor::TVOC", "Sensor::eCO2"]
+datasheets = [
+    "https://www.sciosense.com/wp-content/uploads/2023/12/ENS160-Datasheet.pdf",
+]
+manufacturer = "ScioSense"
+names = ["ENS160"]
+part_numbers = ["ENS160"]
+
+kicad_symbol = ["ENS160-BGLM"]
+
+[[dev_boards]]
+connections = ["Qwiic", "StemmaQt"]
```

Comment:
Since Qwiic and StammeQt are compatible, should we always list both? Or are there any differences I'm missing?